weave-js/src/assets/icons/icon-priority-critical.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M18.5303 10.2197C18.8232 10.5126 18.8232 10.9874 18.5303 11.2803C18.2374 11.5732 17.7626 11.5732 17.4697 11.2803L12 5.81066L6.53033 11.2803C6.23744 11.5732 5.76256 11.5732 5.46967 11.2803C5.17678 10.9874 5.17678 10.5126 5.46967 10.2197L11.4697 4.21967C11.7626 3.92678 12.2374 3.92678 12.5303 4.21967L18.5303 10.2197Z" fill="black"/>
```

Comment:
For most icons we want them to use "currentColor" instead of a specific color like black. (The exception is things like logos where we want fixed colors.) Could you check with design about that for these?

Reply:
Ahh thank you. almost certain this was an oversight.  I'll fix it!

Reply:
Usually the designers will use Moon-800 and then the addicon script will replace it.